docs/install-and-setup.md

```diff
@@ -224,21 +237,37 @@ compinit
 source <(jj util completion zsh)
 ```
 
+dynamic:
+
+```shell
+echo "source <(COMPLETE=zsh jj)" >> ~/.zshrc
+```
+
 ### Fish
 
 ```shell
 jj util completion fish | source
 ```
 
+dynamic:
+
+```shell
+echo "source (COMPLETE=fish jj | psub)" >> ~/.config/fish/config.fish
```

Comment:
In `fish`, `COMPLETE=fish jj | source` is more idiomatic than using `psub`.

I was going to submit a PR, but following Martin's comment, perhaps it's worth collecting them all in one place.

Reply:
(I'm still happy to do a PR if that's easier)

Reply:
I also addressed this here: https://github.com/martinvonz/jj/pull/4847